apps/vmq_diversity/test/mysql_test.lua

```diff
@@ -2,10 +2,9 @@
 config = {
     pool_id = "mysql_test",
     size = 5,
-    user = "vmq_test_user",
-    password = "vmq_test_password",
+    user = "root",
```

Comment:
Runner environment seems to different values.

Reply:
There are no env vars for mysql.